pymess/config.py

```diff
@@ -6,6 +6,8 @@
 
 
 DEFAULTS = {
+    'EMAIL_MESSAGE_ATTACHMENT_MANAGER': 'pymess.models.emails:AttachmentManager',
```

Comment:
Homie, I would rather modify the existing manager to support S3 outta box, man. Nobody wants to mess with custom manager, just because S3 is used, dawg.